<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TLView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEndpoints"/>
   <addaction name="separator"/>
   <addaction name="actionEstablished"/>
   <addaction name="separator"/>
   <addaction name="actionListen"/>
   <addaction name="separator"/>
   <addaction name="actionTime_Wait"/>
   <addaction name="separator"/>
   <addaction name="actionClose_Wait"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionEndpoints">
   <property name="text">
    <string>Endpoints: 0</string>
   </property>
  </action>
  <action name="actionEstablished">
   <property name="text">
    <string>Established: 0</string>
   </property>
  </action>
  <action name="actionListen">
   <property name="text">
    <string>Listen: 0</string>
   </property>
  </action>
  <action name="actionTime_Wait">
   <property name="text">
    <string>Time Wait: 0</string>
   </property>
  </action>
  <action name="actionClose_Wait">
   <property name="text">
    <string>Close Wait: 0</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
